ipt_connect/IPTdev/views.py

```diff
@@ -389,7 +389,7 @@ def physics_fight_detail(request, pfid):
 
 		try:
 			for team in summary_grades:
-				summary_grades[team].append(sum(summary_grades[team]))
+				summary_grades[team].append(sum(summary_grades[team][1:]))
```

Comment:
I'm not sure to understand what was the problem here from your commit message. Could you detail a little bit more?

Also, if this is a bug, do you think it was there for long?

Reply:
The problem was that the reporter's multiplier was added to the sum of points.
This bug appeared tne same time with the summary tables, i.e. after IPT2018, so it affected no tournaments.